prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -41,33 +43,79 @@ private const val sheetName = "LOINC Mapping"
  * LivdTableDownload is the command line interface for the livd-table-download command. It parses the command line
  * for option given as below.
  */
-class LivdTableDownload : CliktCommand(
+class LivdTableDownload() : CliktCommand(
     name = "livd-table-download",
     help = """
-    livd-table downloads the latest LOINC test data, so it can be ingested automatically. 
+    livd-table-download downloads the latest LOINC test data from CDC website, so it can be used to update the 
```

Comment:
This long help string is not allowed.  If you look at the output of ./prime you will see it is cut off.  Make this one sentence and put the rest up with the class description.

Reply:
Done.

Reply:
Done.